maat-court-data-api/src/main/java/gov/uk/courtdata/repository/FinancialAssessmentRepository.java

```diff
@@ -21,4 +21,8 @@
     @Modifying
     @Query(value = "UPDATE FinancialAssessmentEntity fa set fa.replaced = 'Y' WHERE fa.id <> :id AND fa.repId = :repId")
     void updatePreviousFinancialAssessmentsAsReplaced(@Param("repId") Integer repId, @Param("id") Integer id);
+
+    @Query(value = "SELECT count(*) FROM FINANCIAL_ASSESSMENTS fa WHERE fa.REP_ID = :repId AND fa.REPLACED = 'N' AND (fa.VALID IS NULL OR fa.VALID <> 'N') AND (fa.FASS_FULL_STATUS = 'IN PROGRESS' OR fa.FASS_INIT_STATUS  = 'IN PROGRESS')", nativeQuery = true)
+    Long findOutstandingFinancialAssessments(@Param("repId") Integer repId);
+
```

Comment:
is the native query used here because the sql is read-only ?

Reply:
A previous review comment from Matt on PR 401 asked to use native queries to ensure code is consistent